assets/components/ctaLink/ctaLink.jsx

```diff
@@ -31,5 +44,6 @@ export default function CtaLink(props: PropTypes) {
 CtaLink.defaultProps = {
   url: null,
   onClick: null,
+  tabIndex: 0,
```

Comment:
Was this really all that was needed to fix the tab-indexing problem Mike raised?